### PR TITLE
front(docs): update docs for the new playwright setup

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: front/README.md [hunk 1]
```diff
@@ -39,36 +39,16 @@ Launches end to end tests.
 
 It requires:
 
-- Backend containers to be up: `docker compose build core redis editoast postgres`
-- Running front
-- [`poetry`](https://python-poetry.org/docs/#installation) to be installed
```
Comment (shenriotpro) [inline front/README.md:44]: Is poetry no longer needed?

File: front/README.md [hunk 2]
```diff
@@ -39,36 +39,16 @@ Launches end to end tests.
...
-        check=False,
-    )
-```
+- Install playwright dependencies `npx --prefix ./front/ playwright install --with-deps`
+- Backend containers to be up: `TAG=dev docker compose up --no-build --detach redis postgres gateway core editoast`
+- Running front with `TAG=dev docker compose up --build --detach front`
+
+Now you can run the test with `cd front/ && yarn e2e-tests`.
+
+:warning: You may be tempted to launch the frontend locally with `yarn start` but this won't work
```
Comment (shenriotpro) [inline front/README.md]: If not locally, where is the frontend launched? The word may be ambiguous

File: front/README.md [hunk 3]
```diff
@@ -39,36 +39,16 @@ Launches end to end tests.
...
-+           "stdcm-page.spec.ts",
-            "--reporter=line",
-+           "--debug",
-+           "--project=chromium"
-        ],
-        cwd=Path(__file__).parents[2],
-        check=False,
-    )
-```
+- Install playwright dependencies `npx --prefix ./front/ playwright install --with-deps`
```
Comment (shenriotpro) [inline front/README.md]: I found these links to be helpful, can we include them somewhere?
https://classic.yarnpkg.com/lang/en/docs/install/
https://nodejs.org/en/download/package-manager

**Review by shenriotpro — COMMENTED** [review]
Thanks, I managed to run e2e tests for the first time :)

File: front/README.md (hunk 2 above)
Comment (Yohh) [reply front/README.md]: and you can also launch the project on Linux distrib with the `osrd-compose.sh` script, which allows you to use yarn start, maybe we should add a comment about that

**Review by Yohh — COMMENTED** [review]
nice!

should we mention about adding `--ui` or `--debug` options in the script from the `package.json`?

**Review by Yohh — APPROVED** [review]
LGTM!   🐇 💪

File: front/README.md (hunk 1 above)
Comment (woshilapin) [reply front/README.md:44]: It is not indeed. The entire test suite has been changed for frontend tests. Note that you'll still need `poetry` for the global e2e test suite (in `osrd/tests`).

File: front/README.md (hunk 3 above)
Comment (woshilapin) [reply front/README.md]: I added these links in https://github.com/osrd-project/osrd/compare/418a2f76ab65b8a02785c2fcb9d84c2bc18a2402..674abc14156b3e0d455c7a5c7da06fcc54b8b928.

File: front/README.md (hunk 2 above)
Comment (woshilapin) [reply front/README.md]: See https://github.com/osrd-project/osrd/compare/418a2f76ab65b8a02785c2fcb9d84c2bc18a2402..674abc14156b3e0d455c7a5c7da06fcc54b8b928 for the last version, I've also used the nice "Caution blocks" from Github Markdown (just discovered them).

image

**Review by clarani — APPROVED** [review]
LGTM, thanks :white_check_mark:

**Pull request merged**